libraries/AP_AS5600_AOA/AS5600_AOA.cpp

```diff
@@ -0,0 +1,266 @@
+/****************************************************
+ * AMS 5600 Angle of Attack Class for Ardupilot platform
+ * Author: Cole Mero
+ * Date: 15 Dec 2014
+ * File: AS5600_AOA.cpp
+ * Version 1.00
+ * www.ams.com

```

Comment:
Is this right?

Do you need a "based on" pointing to the original source here?